src/lib/components/particles/seat-card/index.jsx

```diff
@@ -0,0 +1,12 @@
+import defaultStyles from './style.module.scss'
+import { mergeStyles } from '$styles/helpers/mergeStyles'
+
+
+
+export function SeatCard({ title, children, styles }) {
```

Comment:
I'm not against having some kind of card component, but I'm not sure its current implementation warrants it.

Reply:
ok I'm leaving the card for now. V happy to abandon it in favour of the same thing happening within the repo.